CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
```

Comment:
Include an entry for the new API method `CaseItemStatementNode::getExpressions`.

Reply:
Good catch.